### DISCUSSION TITLE
Manuscript title

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#87] [General]
Our current title for the manuscript is: 

1. Not just for programmers: A friendly guide on the versatility/benefits of GitHub for accelerating collaborative research in Ecology and Evolution

I want to propose another option and see what else we can brainstorm.

2. Not just for programmers: A friendly guide on how GitHub can accelerate collaborative research in Ecology and Evolution

3. Not just for programmers: A friendly guide to accelerating collaborative research in Ecology and Evolution with GitHub

**Comment by Aguncan** [comment]
I vote for number 2. Adding "how" in title arouse my curiosity.

**Comment by Aariq** [comment]
I also like 2
